plugins/inputs/elasticsearch/elasticsearch.go

```diff
@@ -69,9 +87,13 @@ const sampleConfig = `
   ## within the cluster
   local = true
 
-  ## set cluster_health to true when you want to also obtain cluster level stats
+  ## set cluster_health to true when you want to also obtain cluster health stats
   cluster_health = false
 
+  ## Set cluster_stats to true when you want to also obtain cluster stats from the Master node.
+  ## Currently only implemented when local=true
```

Comment:
I only wanted  cluster stats from the master nodes, and getting the information from just the (local) master seemed somewhat cleaner.  Cluster heath was already set up to not require local, and I did not want to change that.